tomcat/src/test/java/customservlet/MappedExceptionResolversTest.java

```diff
@@ -0,0 +1,41 @@
+package customservlet;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import customservlet.exception.NotFoundExceptionResolverException;
+import nextstep.jwp.presentation.exceptionresolver.NotFoundExceptionResolver;
+import org.apache.coyote.http11.exception.NotFoundException;
+import org.apache.coyote.http11.exception.NotFoundResourceException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class MappedExceptionResolversTest {
+
+    private MappedExceptionResolvers mappedExceptionResolvers;
+
+    @BeforeEach
+    void setUp() {
+        this.mappedExceptionResolvers = MappedExceptionResolvers.getInstance();
+    }
+
+    @Test
+    void 예외클래스를_처리할_Resolver를_찾는다() {
+        // given
+        final NotFoundExceptionResolver notFoundExceptionResolver = new NotFoundExceptionResolver();
+        mappedExceptionResolvers.addResolver(NotFoundException.class, notFoundExceptionResolver);
+
+        // when
+        final ExceptionResolver actual = mappedExceptionResolvers.getResolver(new NotFoundResourceException());
+
+        // then
+        assertThat(actual).isEqualTo(notFoundExceptionResolver);
+    }
+
+    @Test
+    void 예외클래스를_처리할_Resolver가_없으면_예외가_발생한다() {
+        // given, when, then
+        assertThatThrownBy(() -> mappedExceptionResolvers.getResolver(new NotFoundResourceException()))
+                .isInstanceOf(NotFoundExceptionResolverException.class);
```

Comment:
```suggestion
NotFoundResourceException exception = new NotFoundResourceException();
        assertThatThrownBy(() -> mappedExceptionResolvers.getResolver(exception))
                .isInstanceOf(NotFoundExceptionResolverException.class);
```
SonarLint가 띄운 warning인데요, assertThatThrown에 들어갈 람다 표현식에 딱 한가지 로직만 실행될 수 있도록 하기를 권장한다고 합니다. 여러가지 로직이 실행되면 어떠한 것 때문에 예외가 발생하는지 확인하기가 어렵기 때문인 것 같네요!

Reply:
오 그렇군요!! 리펙토링하겠습니다!!